core/arch/arm/kernel/link.mk

```diff
@@ -22,10 +22,16 @@ link-ldadd  = $(LDADD)
 link-ldadd += $(ldflags-external)
 link-ldadd += $(libdeps)
 link-objs := $(filter-out \
+	       $(if $(wildcard $(platform-dir)/link_dummies_paged.c), \
+	            $(out-dir)/$(platform-dir)/link_dummies_paged.o,) \
+	       $(if $(wildcard $(platform-dir)/link_dummies_init.c), \
+	            $(out-dir)/$(platform-dir)/link_dummies_init.o,) \
```

Comment:
The `if`'s are not needed, are they?

Reply:
ok, i'll fix and apply your tag. thanks.

Reply:
i meant, no, they're not and it is simpler :)